### ISSUE TITLE
training error

### ISSUE THREAD

**Issue by liuleiBUAA** [leeyeehoo/CSRNet-pytorch#24]
When I train the model, the loss is nan
Epoch: [0][1170/1200]	Time 0.709 (0.417)	Data 0.020 (0.017)	Loss nan (nan)

**Comment by leeyeehoo** [comment]
Yes. You can make learning rate smaller. like 1e-8 .
